Lab_5/CountriesRecommendation/CountriesRecommendation/Country.cs

```diff
@@ -0,0 +1,29 @@
+namespace CountriesRecommendation {
+    public class Country {
+        public string Name {get;}
+        public int AverageSalary {get;}
+        private long population {get;} 
```

Comment:
Здесь нам больше не нужен геттер (`get;`), поскольку мы пользуемся этим полем класса только внутри класса (гарантировано тем, что оно `private`), а значит всегда будем иметь возможность изменить его как нам необходимо. Если у нас есть полный контроль - дополнительный метод `get` не нужен.
```suggestion
        private long population;
```